scripts/uc-def/uc-def

```diff
@@ -47,10 +47,19 @@ def verify_access_flags(data):
 			raise Exception("'%c' flag of flags '%s' not valid" % (i, data))
 
 def next_useful_line(tag, strip_newline=True, strip_comment=True):
+	if not ('input_lines' in tag):
+		tag['input_lines'] = tag['input_file'].readlines()
```

Comment:
If we can parse the file in stream style, why would we go for read the whole content to memory file.
`file.tell() - this is buggy on windows`  any pointer on the specific problem?
also, `i guess it 1byte ovehead on each line.` if we talk of overhead, we should convert the script in C program.